docs/reference/namespaces.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+title: Namespaces
+parent: pages.reference
+nav_order:  7
+---
+
+# Namespaces
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+![FireFly Namespaces Example](../images/hyperledger-firefly-namespaces-example-with-org.png "FireFly namespaces example")
+
+## Introduction to Namespaces
+
+Namespaces are a construct for segregating data and operations within a FireFly supernode. Each namespace is an isolated environment within a FireFly runtime, that allows independent configuration of:
+
+ * Plugin and infrastructure components
+ * API security
+ * Identity broadcasting
+ * On-chain data indexing
+ * How datatypes, locations of on-chain contrats, etc. should be shared
+
+They can be thought of in two basic modes:
+
+### Multi-party Namespaces
+This namespace is shared with one or more other FireFly nodes. It requires three types of communication plugins - blockchain, data exchange, and shared storage. Organization and node identities must be claimed with an identity broadcast when joining the namespace, which establishes credentials for blockchain and off-chain communication. Shared objects can be defined in the namespace (such as datatypes and token pools), and details of them will be implicitly broadcast to other members.
+
+This type of namespace is used when multiple parties need to share on- and off-chain data and agree upon the ordering and authenticity of that data.
```

Comment:
Worth a link to the multi-party overview doc?

Reply:
Yep, I was holding off on linking immediately because not sure how the sections are going to end up after @aznrayizzle's changes.

Reply:
I think we should link to what's there for now